index.html

```diff
@@ -19,23 +19,170 @@
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section>
+
+  <section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
   </section>
+</section>
+</section>
+<section class="about-me">
+  <h1 class="logo-message">About me</h1>
+  <p class="background">
+    Hello I am a software developer! I can help you build a product,
+    feature or websiteLook through some of my work and experience!
+    If you like what you see and have a project you need coded,
+    do not hestiate to contact me.
+  </p>
+  <button type="button" class="btn">Get My Resume</button>
+  <hr class="line">
+
+  <div class="technology">
+    <img src="image/rectangle.png" alt="parfait image" class="rectangle">
+    <h1 class="heading1">Language</h1>
```

Comment:
- [ ] Kindly change this h1 tag to any hx tag from h2- h6, it is better for your page to have only one h1 tag as the main heading of the page, this will make your project more semantic and easily readable by screen readers 👍🏿